### PR TITLE
Add new keywords and operators

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: contrib/editors/kedit/neon.kld [hunk 1]
```diff
@@ -18,37 +18,35 @@
...
- \(    )
- @@    @@
+ @@"   "@@
 :keyword
 * Language Keywords
  AS
  ASSERT
  BEGIN
  CASE
+ CHOICE
```
Comment (ghewgill) [inline contrib/editors/kedit/neon.kld]: What's CHOICE?

File: contrib/editors/kedit/neon.kld [hunk 2]
```diff
@@ -87,24 +88,18 @@
...
  Dictionary     alternate 6
+ HEXBYTES       alternate 2
  Number         alternate 2
  String         alternate 2
 * NIL is actually a unique value literal, used only with pointer operations.
  NIL            alternate 2
- FIRST          alternate 2
- INDEX          alternate 2
- LAST           alternate 2
- OF             alternate 2
```
Comment (ghewgill) [inline contrib/editors/kedit/neon.kld]: Where did OF go?

File: contrib/editors/kedit/neon.kld [hunk 3]
```diff
@@ -1,5 +1,5 @@
 * neon.kld - KEDIT Language Definition for the Neon Programming Language.
-* Version 1.6 - December, 2015
+* Version 1.6 - May, 2016
```
Comment (ghewgill) [inline contrib/editors/kedit/neon.kld]: What good is having a "version" number if you're not even going to update it?

File: contrib/editors/kedit/neon.kld (hunk 2 above)
Comment (gitlarryf) [inline contrib/editors/kedit/neon.kld]: It got moved to an actual language keyword, rather than a post-processed operator...

**Pull request merged**